assets/xml/objects/object_and.xml

```diff
@@ -1,79 +1,79 @@
 ﻿<Root>
     <File Name="object_and" Segment="6">
```

Comment:
This object documentation looks great, Darkeye is going to ask you to include a comment above this this file section with a brief summary of what this object is used for